sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java

```diff
@@ -215,7 +215,9 @@ public PlannerResult plan() throws SqlParseException, ValidationException, RelCo
       }
       if (parsed.getInsertNode() != null) {
         // Cannot INSERT with BINDABLE.
-        throw e;
+        plannerContext.setPlanningError("Cannot use insert keyword with bindable convention.");
```

Comment:
I think 'bindable convention' is not a user friendly term

Reply:
Thanks, I was planning to use system tables but the comment here [L320](https://github.com/apache/druid/blob/master/sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java#L320) made me think that bindable convention is used for other stuff as well, therefore I was unsure.

Reply:
hmm. this is not even the actual error. The actual error should have been set somewhere already. The fact that we don't use the bindable convention to parse select queries but not insert query, is an internal detail. this particular line should be removed.